src/app/recents/recents.component.html

```diff
@@ -0,0 +1,16 @@
+<app-spinner [loading]="loading"></app-spinner>
+<app-navbar></app-navbar>
+
+<h2 class="recent-title table-title">Recent Artifacts</h2>
+<div class="recent-tabs">
+  <tabset>
+      <tab customClass="recent-table" *ngFor="let type of types" [heading]="type | nodeTypeDisplay" [id]="type">
+        <app-artifacts-table
+          [artifacts]="recentsAll[type]"
+          [tableSpacing]="tableSpacing"
+          [csvName]="csvName"
```

Comment:
Since `csvName` isn't dynamic, it doesn't need to be a variable. How about the following?
```javascript
csvName="recent_artifacts.csv"
```

This also depends on a comment below.